### DISCUSSION TITLE
GraphQL + Glide syntax

### DISCUSSION THREAD

**Discussion by bgauslin** [statamic/cms#4972] [Troubleshooting] [answered]
What is the syntax for Glide image manipulations in a GraphQL query?

I couldn’t find anything in the Statamic v3 documentation, but this is what it looks like for [Craft CMS Image Transforms](https://craftcms.com/docs/3.x/image-transforms.html):

```
image {
  alt: title
  src: url @transform(width: 640, height: 480)
  src_thumb: url @transform(width: 60, height: 60)
}
```

Also, I would like to avoid installing any third-party addons  if possible.

**Comment by duncanmcclean** [comment, marked as answer]
I don't believe Statamic supports GraphQL & Glide together.

**Comment by jasonvarga** [comment]
The docs has an example of using Glide via a custom field.

https://statamic.dev/graphql#custom-fields
